e2e-tests/cypress/integration/case-officer/co-review-questionnaire-status/co-review-questionnaire-status-steps.js

```diff
@@ -15,6 +19,10 @@ goToCaseOfficerPage();
 Then('the page will show the questionnaires with the status {string}',(status)=>{
     selectCaseReferenceFromDb(status);
     cy.get('@caseReference').then((caseReference)=>{
-        getAppealsLink(caseReference).click();
+        let appealSiteAddress = caseReference[25]+", "+caseReference[26]+", "+caseReference[27]+", "+caseReference[28]+", "+caseReference[29];
```

Comment:
Is there a mapping - IDs to human readable word. It will be easier review and debug.
Ignore its going to be a time consuming one to do.